.github/workflows/test.yml

```diff
@@ -10,6 +10,10 @@ jobs:
   test-contracts:
     runs-on: ubuntu-latest
 
+    defaults:

```

Comment:
We need to split out the ui test for other workflow, line 42 here.

Reply:
The UI test is already set to working directory `ui`. Do you mean that?

Reply:
I mean this file has also the ui tests, and it is going to be triggered when has changes in the contract folder https://github.com/fleekxyz/non-fungible-apps/blob/chore/sc-281/change-top-level-folders/.github/workflows/test.yml#L46-L71

Reply:
I think you mean this card https://app.shortcut.com/fleekxyz/story/283/streamline-ci-to-only-do-build-tasks-associated-with-the-changes.  I was going to do it in another PR just because I wanted to research on best workflow files structure before doing that and didn't want to wait to merge this.

But basically I wanted to see what the correct structure is for monorepo. I.e., multiple files for workflows inside the top level .github folder? Or can we add .github inside each top level folder? We can filter as-is but using an external library but I think it's cleaner to do it without that and structuring it correctly. Because at the workflow level, you can filter on folders natively on GHA but not at the job level.

Reply:
I got it, we can split it after, there is no prob.

As far as I know .github needs to be in the root folder, so I think it will have multiple workflow files at the root level.